packages/go_router/test/delegate_test.dart

```diff
@@ -89,28 +89,6 @@ void main() {
     );
   });
 
-  group('canPop', () {
```

Comment:
can you add some test for canpop too?

Reply:
8749762991346f837d126bb9bdf4483810155c40